src/validation.cpp

```diff
@@ -3116,10 +3116,12 @@ bool CheckBlock(const CBlock& block, CValidationState& state, const Consensus::P
             return state.DoS(100, false, REJECT_INVALID, "bad-cb-multiple", false, "more than one coinbase");
 
     // Check transactions
-    for (const auto& tx : block.vtx)
-        if (!CheckTransaction(*tx, state, true))
-            return state.Invalid(false, state.GetRejectCode(), state.GetRejectReason(),
-                                 strprintf("Transaction check failed (tx hash %s) %s", tx->GetHash().ToString(), state.GetDebugMessage()));
+    for (const auto& tx : block.vtx) {
+        if (!CheckTransaction(*tx, state, true)) {
+            LogPrintf("Transaction check failed (tx hash %s) %s\n", tx->GetHash().ToString(), state.GetDebugMessage());
```

Comment:
In commit "Check transactions just logs a message" (94c2cdb88049af5283a7c1f52ea6e52ac2946686)

Could you update the commit message to say whether this commit changes behavior at all, and what the motivation is? At first glance it seems like this probably doesn't change behavior, and the only motivation is to simplify code. But I could easily be missing something.

Reply:
Done

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15141#discussion_r250800408

> Done

Thanks, new commit is "Remove redundant state.Invalid() call after CheckTransaction()" (59ff8e67c2c62ec11d76d3d1b54dc4829363ad5e)